docker-compose.yml

```
@@ -0,0 +1,48 @@
+version: '3'
+services:
+
+  db:
+    image: postgres:10-alpine
+    ports:
+      - "5432:5432"
+    environment:
+      POSTGRES_DB: app_db
+      POSTGRES_USER: app_user
+      POSTGRES_PASSWORD: postgres!secret
+  # Uncomment this if you want to persist the data.
+  # volumes:
+  #   - "./pgdata:/var/lib/postgresql/data"
```

Persistence should be the default, it's also important to nest it in a folder. postgresql sets ownership on the directory and it will be impossible to dockerignore without nesting the folder and ignoring the parent folder.
```suggestion
    volumes:
      - ./data/postgres:/var/lib/postgresql/data
```